From 34fb0785f1d66ba1f79b470223426031f6258782 Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Wed, 17 Sep 2014 17:48:54 +0100
Subject: [PATCH 1/3] iprutils: Don't use gettext

... since we don't have any translation.

Based on the former patch by Jeremy Kerr.

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
---
 iprconfig.h |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/iprconfig.h b/iprconfig.h
index cad591a..4a1fa0d 100644
--- a/iprconfig.h
+++ b/iprconfig.h
@@ -11,9 +11,7 @@
  *
  **/
 
-#include <libintl.h>
-
-#define _(string) gettext(string)
+#define _(string) (string)
 #define __(string) (string)
 #define EXIT_FLAG		0x8000	/* stops at given screen on exit call */
 #define CANCEL_FLAG		0x4000	/* stops at given screen on quit call */
-- 
1.7.1

